src/components/routeComponents/memberInformationGrid/attendanceResportModel/AttendanceReportModel.jsx

```diff
@@ -0,0 +1,89 @@
+import React, { useState } from 'react';
+import styled from 'styled-components';
+import Modal from 'pepcus-core/lib/Modal';
+import Box from 'pepcus-core/lib/Box';
+import ModalHeader from 'components/routeComponents/memberInformationGrid/attendanceResportModel/ModelHeader';
+import Accordion from 'components/common/Accordion/Accordion';
+import AttendanceDisplayCard from 'components/routeComponents/memberInformationGrid/attendanceResportModel/AttendanceDisplayCard';
+
+const AttendanceReportModel = () => {
+	const [isModelOpen, setIsModelOpen] = useState(false);
+
+	const openModel = () => {
+		setIsModelOpen(true);
+	};
+
+	const closeModal = () => {
+		setIsModelOpen(false);
+	};
+
+	const renderModel = () => {
+		if (isModelOpen) {
+			return (
+				<Modal open={isModelOpen} style={{ padding: '0' }}>
+					<StyledBox
+						backgroundColor="modal"
+						padding="0"
+						borderColor="tertiary"
+						margin="0"
+						width="auto">
+						<ModalHeader onclick={closeModal} />
+						<StyledModelBody>
+							<Accordion>
+								<AttendanceDisplayCard />
+							</Accordion>
+						</StyledModelBody>
+					</StyledBox>
+				</Modal>
+			);
+		}
+		return null;
+	};
+	return (
+		<>
+			<StyledModelLink onClick={openModel}>Attendance Model</StyledModelLink>
```

Comment:
Get these strings from JSON constants.
```suggestion
			<StyledModelLink onClick={openModel}>Attendance Report</StyledModelLink>
```

Reply:
Resolved